### PR TITLE
ED-2085:  OCI support for Build/Lern/LernFlinkJobs

### PR DESCRIPTION
**Pull request by ddevadat**
Build/Lern/LernFlinkJobs
Added oci support for Build/Lern/LernFlinkJobs

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lms-jobs/relation-cache-updater/src/main/scala/org/sunbird/job/relationcache/functions/RelationCacheUpdater.scala
Comment (Hari-stackroute) [inline lms-jobs/relation-cache-updater/src/main/scala/org/sunbird/job/relationcache/functions/RelationCacheUpdater.scala:1]: Seems this file contains unnecessary imports and line no: 31 doesn't requires comma.

**Review by Hari-stackroute — CHANGES_REQUESTED** [review]
I request one file change, apart from everything looks fine.

File: lms-jobs/relation-cache-updater/src/main/scala/org/sunbird/job/relationcache/functions/RelationCacheUpdater.scala [hunk 1]
```diff
@@ -25,7 +28,7 @@ class RelationCacheUpdater(config: RelationCacheUpdaterConfig)
     private[this] val logger = LoggerFactory.getLogger(classOf[RelationCacheUpdater])
     private var dataCache: DataCache = _
     private var collectionCache: DataCache = _
-    lazy private val mapper: ObjectMapper = new ObjectMapper()
+    lazy private val mapper: ObjectMapper = new ObjectMapper();
```
Comment (AmiableAnil) [inline lms-jobs/relation-cache-updater/src/main/scala/org/sunbird/job/relationcache/functions/RelationCacheUpdater.scala:31]: Optimize imports and remove the semicolon. Semicolon is not required in scala.

File: lms-jobs/credential-generator/collection-certificate-generator/src/test/scala/org/sunbird/job/certgen/spec/CertificateGeneratorFunctionTest.scala [hunk 2]
```diff
@@ -34,7 +34,7 @@ class CertificateGeneratorFunctionTest extends BaseTestSpec {
   lazy val jobConfig: CertificateGeneratorConfig = new CertificateGeneratorConfig(config)
   val httpUtil: HttpUtil = new HttpUtil
   val mockHttpUtil = mock[HttpUtil](Mockito.withSettings().serializable())
-  val storageParams: StorageParams = StorageParams(jobConfig.storageType, jobConfig.storageKey, jobConfig.storageSecret, jobConfig.containerName)
+  val storageParams: StorageParams = StorageParams(jobConfig.storageType, jobConfig.storageKey, jobConfig.storageSecret, jobConfig.containerName,Option(jobConfig.storageEndpoint))
```
Comment (AmiableAnil) [inline lms-jobs/credential-generator/collection-certificate-generator/src/test/scala/org/sunbird/job/certgen/spec/CertificateGeneratorFunctionTest.scala:37]: To make the code readable, please put the space after comma. Same for other files as well.

File: jobs-distribution/Dockerfile [hunk 3]
```diff
@@ -7,6 +7,8 @@ RUN apt-get install -y imagemagick
 COPY target/jobs-distribution-1.0.tar.gz /tmp
 USER flink
 RUN tar -xvf /tmp/jobs-distribution-1.0.tar.gz -C $FLINK_HOME/lib/
+# RUN cp $FLINK_HOME/opt/flink-s3-fs-hadoop-1.13.5.jar $FLINK_HOME/lib/flink-s3-fs-hadoop-1.13.5.jar
```
Comment (AmiableAnil) [inline jobs-distribution/Dockerfile:10]: Remove the commented line.

**Pull request merged**